presto-druid/src/test/java/com/facebook/presto/druid/TestDruidConfig.java

```diff
@@ -33,7 +33,8 @@ public void testDefaults()
                 .setDruidCoordinatorUrl(null)
                 .setDruidSchema("druid")
                 .setComputePushdownEnabled(false)
-                .setHadoopConfiguration(""));
+                .setHadoopConfiguration("")
+                .setDruidAuthenticationType(null));
```

Comment:
should it be DruidAuthenticationType.NONE?

Reply:
changed to DruidAuthenticationType.NONE